### PR TITLE
Questions Screen Style & Mobile View

### PR DESCRIPTION
**Pull request by sonamgupta21**
Commented the fake data.
image
image

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -7,50 +7,13 @@ $(document).ready(() => {
...
-      data: data,
-    });
-
+    const currentUrl = window.location.href;
+    if (currentUrl.includes("guides.html")) {
+      renderGuidesList();
+    }
+    // else if (currentUrl.includes("questions.html")) {
+    //   renderQuestionsList();
+    // }
```
Comment (nbhavana) [inline js/main.js:16]: Remove the commented code

File: js/main.js [hunk 2]
```diff
@@ -150,4 +113,68 @@ function newsletterFormHandler(evt) {
...
\ No newline at end of file
+}
+
+function renderGuidesList(){
+  let titleList = ["ElasticSearch", "Logs"];
+  let guidesData = [
+    {
+      title: "ElasticSearch",
+      // basicIntro:"Basic intro",
+      detailedList: [
```
Comment (nbhavana) [inline js/main.js:124]: remove the commented line

File: js/main.js [hunk 3]
```diff
@@ -150,4 +113,68 @@ function newsletterFormHandler(evt) {
...
+              redirectPage: "../guides/elasticSearch.html",
+            }
+          ],
+        }
+      ],
+    },
+    {
+      title: "2222",
+      // basicIntro: "Basic intro",
+      detailedList: [
```
Comment (nbhavana) [inline js/main.js:140]: same here

**Review by nbhavana — APPROVED** [review]
few nit-piks. Once resolved you can merge the PR

File: js/main.js (hunk 1 above)
Comment (sonamgupta21) [reply js/main.js:16]: This part removes the fake data for questions. I'll leave it as is since it helps to understand the format of the questions data.

**Pull request merged**